src/sardana/pool/poolmeasurementgroup.py

```diff
@@ -1384,3 +1386,18 @@ def stop(self):
     def abort(self):
         self._pending_starts = 0
         PoolGroupElement.abort(self)
+
+    # -------------------------------------------------------------------------
+    # utils
+    # -------------------------------------------------------------------------
+
+    def _verify_controllers(self, cfg):
+        pool = self.pool
+        for ctrl_name, _ in list(cfg['controllers'].items()):
```

Comment:
No real point in using `.items()` here, and since you're not changing `cfg["controllers"]` you can just:
```python
for ctrl_name in ctrl["controllers"]:
```